GDSC-Unijos/Health-App-Backend
# scheduling

How to Test:

1. `Authorize:` http://localhost:3001/authorize 
2. `event scheduling:` `http://localhost:3001/events/schedule-event`
3. `list events:` `http://localhost:3001/events/list-events`

delete the `token.json` file before running the authorization url. 

wahala dey sha. check for the executability and return me feedback.
light no still dey ohhh. somewhay...

return subsequent observation after a tour of the codes added. thanks
